scripts/traning.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+
+# BillDiction Library
```

Comment:
Warum steht da BillDiction Library?

Reply:
Because I copied this from another project.